### PR TITLE
Fixes a couple of bugs and warnings.

### PR DESCRIPTION
**Pull request by eet349**
Description
There were a couple of warnings that were cleared up. Also added a spinner to the RenderAlerts component to be shown while the clubs and alerts are being fetched.
What work was done?  
added a key prop to the clubs in ViewClubs.
removed strict mode
Added a spinner while clubs are being fetched
Why was this work done?  
The warnings could possibly cause the deploy to fail

[] New feature (non-breaking change which adds functionality)
Change Status  
[x ] Completed, ready to review and merge  
Has This Been Tested  
[ ] Yes
[X] No

Checklist
[X] My code follows the style guidelines of this project  
[X] I have performed a self-review of my own code  
[X] My code has been reviewed by at least one peer  
[] I have commented my code, particularly in hard-to-understand areas  
[] I have made corresponding changes to the documentation  
[X] My changes generate no new warnings  
[X] There are no merge conflicts

### CODE REVIEW

**Review by ashtilawat23 — APPROVED** [review]

File: src/components/pages/Alerts/RenderAlerts.js [hunk 1]
```diff
@@ -30,8 +33,8 @@ function RenderAlerts(props) {
         </Sider>
         <Content>
           {context.clubs.length === 0 ? (
```
Comment (L-Steinmacher) [inline src/components/pages/Alerts/RenderAlerts.js:35]: Thata some fine loader placement.

File: src/components/pages/Clubs/ViewClubs.js [hunk 2]
```diff
@@ -57,6 +57,7 @@ function ViewClubs(props) {
     context.clubs.forEach(club => {
       const newRow = {
         clubName: club.clubname,
+        key: club.clubid,
```
Comment (L-Steinmacher) [inline src/components/pages/Clubs/ViewClubs.js:60]: Thank you for adding the key!

File: src/index.js [hunk 3]
```diff
@@ -47,9 +47,7 @@ import HiddenRoute from './utils/HiddenRoute';
 
 ReactDOM.render(
   <Router>
-    <React.StrictMode>
-      <App />
-    </React.StrictMode>
+    <App />
```
Comment (L-Steinmacher) [inline src/index.js:50]: Removing strict mode...  Mhhmmm

**Review by L-Steinmacher — APPROVED** [review]
This code very clearly fixes some bugs and warnings!

**Pull request merged**